core/src/test/java/org/apache/iceberg/TestMetadataTableFilters.java

```diff
@@ -447,6 +441,12 @@ public void testPartitionSpecEvolutionRemovalV2() {
     assertThat(tasks).hasSize(expectedScanTaskCount(3));
   }
 
+  private DeleteFile posDelete(Table table, DataFile dataFile) {
```

Comment:
Do we need to pass `Table` or can we just use a similar method from `TestBase`?

Reply:
I'll double check that and will clean that up in the next PR